gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
-#Fri Apr 08 18:13:41 PDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
```

Comment:
do we need to change it also for toothpick-sample/gradle-wrapper?

Reply:
We do. Sorry that I didn't realize that there are two root projects under the same repo so I didn't update gradle for the sample project. Fixed. You probably need to run CI again.